react/week4/github-user-app/src/components/UserContext.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, createContext } from "react";
+export const UserContext = createContext();
+
+export const UserProvider = ({ children }) => {
+    const [isLoading, setIsLoading] = useState(false);
+    const [users, setUsers] = useState([]);
+    const [userSearch, setUserSearch] = useState("");
+    const [error, setError] = useState("");
+    const [repos, setRepos] = useState([]);
+
+    const fetchUsers = async (url) => {
+        try {
+            const fetchUsers = await fetch(url)
+            const usersData = await fetchUsers.json();
+            console.log(usersData)
```

Comment:
I would usually remove `console.log`s before pushing my code, so the end-user does not see data in their console